pkg/apis/config/v1beta1/types.go

```diff
@@ -73,3 +74,20 @@ type CapacitySchedulingArgs struct {
 	// KubeConfigPath is the path of kubeconfig.
 	KubeConfigPath *string `json:"kubeconfigpath,omitempty"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// TargetLoadPackingArgs holds arguments used to configure TargetLoadPacking plugin.
+type TargetLoadPackingArgs struct {
+	metav1.TypeMeta 	`json:",inline"`
+
+	// Node target CPU Utilisation for bin packing
+	TargetCPUUtilization float64 `json:"mode,omitempty"`
+	// Default CPU requests to use for best effort QoS
+	DefaultCPURequests   int64 `json:"mode,omitempty"`
```

Comment:
In a versioned-types, we should pointer, int64->*int64

Reply:
I see, what's the reason for this?

Reply:
For CPU Requests it is better to use `v1.ResourceList`